### ISSUE TITLE
Question: How did you support Tablet version ?

### ISSUE THREAD

**Issue by deadcoder0904** [RCiesielczuk/HackerBuzz-ReactNative#1]
First of all, thank you. It looks really cool. Love it ❤️ 

So my question is how did you support the Tablet Version ? Also, I want to know when you developed this application, did you keep iPhone & iPad simulators open or did you first make it on mobile then adjusted everything to support tablet. 

Please give a detailed response as I want to make my application for Tablet but Flexbox doesn't help everywhere. Any links or tuts will be much appreciated. Thanks 👍

**Comment by deadcoder0904** [comment]
Dude, again the app is so cool. I think this is the best app out there. Absolutely in love with UI 😍

**Comment by a-dilettante** [comment]
Hi, 

Thank you for your feedback, really appreciate it. When I initially started working on this project I've been testing it on an iPhone and an Android phone. I never really focused on tablets, but I agree that the app needs to be optimized to support them. I ran the app across simulators to see what it would look like on tablets, but that's about it.

I'll try and update the roadmap this week and see where this should fit in.

**Comment by deadcoder0904** [comment]
I saw Tablet screenshots in Play Store, so its already done right ❔

**Comment by a-dilettante** [comment]
Yeah, both apps support tablets, but nothing special has been done for them. All the work I've done for phones is reflected for the tablets.
